clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/HostLevelIngestionStats.java

```diff
@@ -253,40 +249,42 @@ public HostLevelIngestionStats(
     // Register an aggregate metric for disk_usage_in_bytes metric
     final boolean isTotalStats = isTotalStats();
     registerSensor(
-        "disk_usage_in_bytes",
-        new Gauge(
-            () -> ingestionTaskMap.values()
+        new AsyncGauge(
+            (ignored, ignored2) -> ingestionTaskMap.values()
                 .stream()
                 .filter(task -> isTotalStats ? true : task.getStoreName().equals(storeName))
                 .mapToLong(
                     task -> isTotalStats
                         ? task.getStorageEngine().getCachedStoreSizeInBytes()
                         : task.getStorageEngine().getStoreSizeInBytes())
-                .sum()));
+                .sum(),
+            "disk_usage_in_bytes"));
```

Comment:
Why is the ordering of the params swapped? Is there a new `registerSensor` function? Why not keep the previous ordering?

Reply:
It is not really swapped since `AsyncGauge` requires two params:
```
public AsyncGauge(Measurable measurable, String metricName)
```

And
```
protected Sensor registerSensor(NamedMeasurableStat... stats)
```
doesn't need sensorName anymore since the params contains sensorName.

Reply:
Ok I see. I guess it would have been more aesthetically pleasing if `AsyncGauge` took the name as the first param, but that's in Tehuti so I guess that ship has sailed. Carry on then 🤗